cohttp.install

```diff
@@ -1,7 +1,16 @@
 bin: [
-  "?cohttp-server-async"
-  "?cohttp-server-lwt"
-  "?cohttp-curl-async"
-  "?cohttp-curl-lwt"
-  "?cohttp-proxy-lwt"
+  "?cohttp_server_async.native" { "cohttp-server-async" }
+  "?cohttp_server_async.byte"   { "cohttp-server-async" }
```

Comment:
does the ordering matter? I'd prefer to install native code ones rather than bytecode.